example/minimal/main.go

```diff
@@ -29,15 +31,26 @@ func (s *Server) registerUser(schema *schemabuilder.Schema) {
 func (s *Server) registerQuery(schema *schemabuilder.Schema) {
 	object := schema.Query()
 
-	object.FieldFunc("users", func(ctx context.Context) ([]*User, error) {
+	var tmp messageEnumType
+	schema.EnumReg(tmp, map[string]interface{}{
```

Comment:
maybe schema.Enum instead of schema.EnumReg, so it matches schema.Object and schema.Query?